src/Core/Scheduler/Providers/TileProvider.js

```diff
@@ -32,6 +34,100 @@ TileProvider.prototype.preprocessDataLayer = function preprocessLayer(layer, vie
         throw new Error(`Cannot init tiled layer without schemeTile for layer ${layer.id}`);
     }
 
+    layer.addColorLayer = (colorLayer, addToView = true) => {
```

Comment:
you need the documentation

Reply:
Done see: https://github.com/iTowns/itowns/pull/546/files#diff-61e012c3e00bdc510be33d9eac684b2aR193